lib/Analysis/ScalarEvolution.cpp

```diff
@@ -88,6 +88,8 @@
 #include "llvm/Support/ErrorHandling.h"
 #include "llvm/Support/MathExtras.h"
 #include "llvm/Support/raw_ostream.h"
+#include "dxc/HLSL/DxilValueCache.h" // HLSL Change
```

Comment:
With this change, LLVMAnalysis now depends on LLVMHLSL.

To fix the Linux builds, add HLSL to the list of required_libraries in lib/Analysis/LLVMBUild.txt